src/sentry/static/sentry/app/components/smartSearchBar.jsx

```diff
@@ -672,6 +726,20 @@ const SmartSearchBarContainer = withApi(
   )
 );
 
+const PinButton = styled(Button)`
+  margin-right: ${space(0.5)};
+  position: absolute;
+  right: 26px;
+  top: 10px;
+`;
+
+const PinIcon = styled(InlineSvg)`
+  color: ${p => (p.isPinned ? 'yellow' : 'gray')};
```

Comment:
Should we use theme colors?

Reply:
There's going to be follow ups with design updates that will change this

Reply:
Had to make some changes anyway so cleaned this up